clientGQL_test.go

```diff
@@ -97,6 +97,27 @@ func ATestClient(t *testing.T, endpoint string) *ol.Client {
 		autopilot.GraphQLQueryFixtureValidation(t, fmt.Sprintf("%s_request.json", endpoint)))))
 }
 
+type TestRequest struct {
+	Request  string
+	Response string
+}
+
+func RegisterPaginatedEndpoint(t *testing.T, endpoint string, requests ...TestRequest) string {
+	url := fmt.Sprintf("/LOCAL_TESTING/%s", endpoint)
+	requestCount := 0
+	autopilot.Mux.HandleFunc(url, func(w http.ResponseWriter, r *http.Request) {
+		GraphQLQueryTemplatedValidation(t, requests[requestCount].Request)(r)
+		TemplatedResponse(requests[requestCount].Response)(w)
+		requestCount += 1
```

Comment:
This count pair'd with the variadic function param `requests` allow us to create different response for each subsequent request - in the case of pagination the 1st request will return page info has next page == true and the 2nd will return a response where it is false.